src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -1033,5 +1036,27 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
     }
 }
 
+const applyQueryParams = (
+    query: Knex.QueryBuilder,
+    queryParams: IQueryParam[],
+): void => {
+    queryParams.forEach((param) => {
+        switch (param.operator) {
+            case 'IS':
+                query.where(param.field, '=', param.value);
+                break;
+            case 'IS_NOT':
+                query.where(param.field, '!=', param.value);
+                break;
+            case 'IS_ANY_OF':
+                query.whereIn(param.field, param.value as string[]);
+                break;
+            case 'IS_NOT_ANY_OF':
```

Comment:
IS_NOT_IN would sound more natural to me. Or even `NOT_IN`. I think we should keep  domain language close to SQL. Same applies to the UI. What do you think @nicolaesocaciu ?

Reply:
I think we following language based on UI elements and those were taken 1 to 1 from linear. Maybe they have thought about it more than us.

Reply:
Either way we go, I think frontend should have same texts.

Reply:
I understand both expressions but for me it's a bit more clear when is see **"Is not any of"** compared with **"Is not in / Not in"**

I will ask also the team what they think, and we will have at some point some user testing with the filters and i will try to see what is the perception about these expressions